### PR TITLE
Removing old network stack (NetworkSDK)

### PR DESCRIPTION
**Pull request by bhariharan**
I've moved it to another repo internally for now. Everything is compiling.

### CODE REVIEW

File: SalesforceSDKCore.podspec [hunk 1]
```diff
@@ -49,7 +49,7 @@ Pod::Spec.new do |s|
...
...alesforceSDKCore/SalesforceSDKCore/Classes/Util/NSURL+SFStringUtils.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFApplicationHelper.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFDirectoryManager.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFJsonUtils.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFManagedPreferences.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFPreferences.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFRootViewManager.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKCoreLogger.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKResourceUtils.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKSoqlBuilder.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKSoslBuilder.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKSoslReturningBuilder.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKWebUtils.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SalesforceSDKCoreDefines.h','libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/UIColor+SFColors.h','libs/SalesforceSDKCore/SalesforceSDKCore/SalesforceSDKCore.h'
```
Comment (bhariharan) [inline SalesforceSDKCore.podspec]: Removed network related files

File: libs/SalesforceSDKCore/Configuration/SalesforceSDKCore-Common-Dynamic.xcconfig [hunk 2]
```diff
@@ -10,6 +10,3 @@
 
 INSTALL_PATH = /
 SKIP_INSTALL = YES
-
-MODULEMAP_FILE = $(SRCROOT)/SalesforceSDKCore/SalesforceSDKCore.modulemap
```
Comment (bhariharan) [inline libs/SalesforceSDKCore/Configuration/SalesforceSDKCore-Common-Dynamic.xcconfig]: The only reason we had a custom module map was for `CSFPrivateDefines.h`. We don't need this anymore.

File: libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -90,13 +90,6 @@
 		4FD235DA1EFD8026004747DD /* SFAuthenticationSafariControllerHandler.h in Headers */ = {isa = PBXBuildFile; fileRef = 4FD235D81EFD7FB8004747DD /* SFAuthenticationSafariControllerHandler.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		4FE5332A1BFFE70600814D2A /* Main_iPad.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 4F7EB4981BFFCEF600768720 /* Main_iPad.storyboard */; };
 		4FE5332B1BFFE70600814D2A /* Main_iPhone.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 4F7EB49A1BFFCEF600768720 /* Main_iPhone.storyboard */; };
-		6F53B1711CE3A7B100AD81B8 /* SFLoggerMacros.h in Headers */ = {isa = PBXBuildFile; fileRef = 6F53B1701CE3A7B100AD81B8 /* SFLoggerMacros.h */; settings = {ATTRIBUTES = (Public, ); }; };
```
Comment (bhariharan) [inline libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj]: `SFLogger` went with `NetworkSDK`.

**Comment by huminzhi** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Code coverage under minimum of 80%</td>
    </tr>
  </tbody>
</table>

## Current coverage for SalesforceSDKCore.framework is `47%`
Files changed | - | - 
--- | --- | ---
NSArray+SFAdditions.m | `0%` | :skull:

---
## Current coverage for SalesforceAnalytics.framework is `42%`
✅ *No files affecting coverage found*

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_CI-result">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by bhariharan** [comment]
Everything is compiling, tests are passing. This PR is really just a large-scale deletion of files. The only changes I've made:
1. Tweaked a test class that was using a mock `CSFNetwork` and mock `CSFAction`.
2. Removed the custom LLVM module maps that were used.
3. Updated pod specs.
4. Updated umbrella headers.

**Pull request merged**